lucille-core/src/main/java/com/kmwllc/lucille/stage/AddRandomBoolean.java

```diff
@@ -20,24 +21,45 @@
  */
 public class AddRandomBoolean extends Stage {
 
-  private final String fieldName;
-  private final int percentTrue;
+  public static class AddRandomBooleanConfig extends BaseStageConfig {
+
+    private String fieldName = "data";
+    private int percentTrue = 50;
+
+    public void apply(Config config) {
+      fieldName = ConfigUtils.getOrDefault(config, "field_name", fieldName);
+      percentTrue = ConfigUtils.getOrDefault(config, "percent_true", percentTrue);
+    }
+
+    public void validate() throws StageException {
+      // Call super.validate(); if BaseStageConfig implements it
+      if (percentTrue > 100 || percentTrue < 0) {
+        throw new StageException("Invalid value for percent_true. Must be between 0 and 100 inclusive.");
+      }
+    }
+  
+    public String getFieldName() { return fieldName; }
+    public int getPercentTrue() { return percentTrue; }
+
+  }
+
+  AddRandomBooleanConfig config;
 
-  public AddRandomBoolean(Config config) throws StageException {
-    super(config, Spec.stage()
+  public AddRandomBoolean(Config configIn) throws StageException {
+    super(configIn, Spec.stage()
```

Comment:
it'd be nice if we could create the stage spec from the stageconfig class so it could become boiler plate instead of having to define the required/optional fields.